gui/src/components/Layout.tsx

```diff
@@ -44,7 +45,7 @@ const Layout = () => {
   const navigate = useNavigate();
   const location = useLocation();
   const dispatch = useAppDispatch();
-  const onboardingCard = useOnboardingCard();
+  const [showGraniteOnboardingCard, setShowGraniteOnboardingCard] = useState<boolean>(window.showGraniteCodeOnboarding??false);
```

Comment:
Do you have prettier installed and vscode set to format on save? this looks way past where prettier would wrap a line.

Reply:
done